mono/metadata/w32file-unix.c

```diff
@@ -14,6 +14,12 @@
 #ifdef HAVE_SYS_STATVFS_H
 #include <sys/statvfs.h>
 #endif
+#ifdef HAVE_COPYFILE_H
+#include <copyfile.h>
+#  if !defined(COPYFILE_CLONE)
```

Comment:
`#ifndef` and no indentation, for consistency with rest of file.

Reply:
No - that is a bad style that someone went and added in recent years.   Undo that disaster please.